app/scripts/lib/constants.js

```diff
@@ -78,7 +78,12 @@ define([], function () {
     ACCESS_TYPE_ONLINE: 'online',
     ACCESS_TYPE_OFFLINE: 'offline',
 
-    DEFAULT_XHR_TIMEOUT_MS: 2500
+    DEFAULT_XHR_TIMEOUT_MS: 2500,
+
+    EVENTS: {
```

Comment:
Maybe a new module, event_names or something, and we can rip some out elsewhere?